ctapipe/tools/stage1.py

```diff
@@ -77,12 +77,17 @@ class Stage1Tool(Tool):
     def setup(self):
 
         # setup components:
-        self.event_source = EventSource(parent=self)
+        self.event_source = EventSource.from_config(parent=self)
```

Comment:
Same as above, please don't just change things that are supposed to work without creating an issue if they don't